src/api/WebApi/WebApi/Controllers/WildBoarIotDataController.cs

```diff
@@ -15,9 +19,39 @@ public WildBoarIotDataController(IWildBoarIotDataService _wildBoarIotDataService
             wildBoarIotDataService = _wildBoarIotDataService;
         }
 
-        [HttpGet()]
-        public async Task<List<WildBoarIotData>> Get() =>
-            await wildBoarIotDataService.GetAsync();
+        [HttpGet("{sort?}/{format?}/{type?}/{weight?}/{occupied?}/{date_start?}/{date_end?}")]
+        public async Task<List<WildBoarIotData>> Get(string? sort, string? format, string? type, int weight,
```

Comment:
For every `String` inside this function, please align case of letters at the beginning, because it is tricky and different for every field in record (`type` is in CAPS, but `sort` is in small letters). For example, there should be no difference if someone provides: `type=KLATA_MACHINE` and `type=klata_machine`. I personally would align this to ALL CAPS letters.